### PR TITLE
Update elections documentation

### PR DESCRIPTION
**Pull request by ldmberman**

### CODE REVIEW

**Comment by ldmberman** [comment]
Currently based on the other branch of mine. Relevant changes are limited to `docs/server/source/server-reference/bigchaindb-cli.md`.

File: docs/server/source/server-reference/bigchaindb-cli.md [hunk 1]
```diff
@@ -83,107 +83,113 @@ configuration file as documented under
...
+###### election new chain-migration
 
-Create an election to halt block production, to allow for a version change across breaking changes.
+Create an election to halt block production, to coordinate on making a Tendermint upgrade with a backwards-incompatible chain.
 
 
 ```bash
-$ bigchaindb election new migration --private-key PATH_TO_YOUR_PRIVATE_KEY
-[SUCCESS] Submitted proposal with id: <election_id>
+$ bigchaindb election new chain-migration --private-key <path-to-theprivate-key>
```
Comment (ttmc) [inline docs/server/source/server-reference/bigchaindb-cli.md]: Minor typo: `theprivate-key>` should be `the-private-key>`

File: docs/server/source/server-reference/bigchaindb-cli.md [hunk 2]
```diff
@@ -83,107 +83,113 @@ configuration file as documented under
...
 ```
 
-- `--private-key`: The path to Tendermint's private key which can be generally found at `/home/user/.tendermint/config/priv_validator.json`.
+- `<path-to-the-private-key>` is the path to the private key of the validator who proposes the election. Tendemint places it at  `.tendermint/config/priv_validator.json`.
```
Comment (ttmc) [inline docs/server/source/server-reference/bigchaindb-cli.md]: Tendemint --> Tendermint, both here and elsewhere. (Do a search for "Tendemint".)

File: docs/server/source/server-reference/bigchaindb-cli.md [hunk 3]
```diff
@@ -83,107 +83,113 @@ configuration file as documented under
...
 $ bigchaindb election new migration --private-key /home/user/.tendermint/config/priv_validator.json
 [SUCCESS] Submitted proposal with id: 04a067582cf03eba2b53b82e4adb5ece424474cbd4f7183780855a93ac5e3caa
 ```
 
-**NOTE** `migration` elections will halt block production at whichever blockheight they are approved. Once the election is concluded, the validators will need to restart their systems with a new `chain_id` to resume normal operations.
+Concluded chain migration elections halt block production at whichever block height they are approved.
+Afterwards, validators are supposed to upgrade Tendermint, set new `chain_id`, `app_hash`, and `validators` (to learn these values, use the [election show](#election-show) command) in `genesis.json`, make and save a MongoDB dump, and restart the system.
 
 
+For more details about how chain migrations work, refer to [BEP-42](https://github.com/bigchaindb/BEPs/tree/master/42).
```
Comment (ttmc) [inline docs/server/source/server-reference/bigchaindb-cli.md]: Maybe we could be a bit more helpful and say "refer to [Type 3 scenarios in BEP-42](https://github.com/bigchaindb/BEPs/tree/master/42#type-3-scenarios)."

File: docs/server/source/server-reference/bigchaindb-cli.md [hunk 4]
```diff
@@ -83,107 +83,113 @@ configuration file as documented under
...
+
+Note that election proposers do not automatically approve elections by proposing them.
 
+For more details about how validator set changes work, refer to [BEP-21](https://github.com/bigchaindb/BEPs/tree/master/21).
 
-###### election new migration
+#### election new chain-migration
 
-Create an election to halt block production, to allow for a version change across breaking changes.
+Create an election to halt block production, to coordinate on making a Tendermint upgrade with a backwards-incompatible chain.
```
Comment (kansi) [inline docs/server/source/server-reference/bigchaindb-cli.md]: I would be good to bold `backwards-incompatible` chain

**Pull request merged**